Source/.editorconfig

```diff
@@ -0,0 +1,91 @@
+[*.{cpp,h}]
+
+# Naming convention rules (note: currently need to be ordered from more to less specific)
+
+cpp_naming_rule.aactor_prefixed.symbols						= aactor_class
+cpp_naming_rule.aactor_prefixed.style						= aactor_style
+
+cpp_naming_rule.swidget_prefixed.symbols					= swidget_class
+cpp_naming_rule.swidget_prefixed.style						= swidget_style
+
+cpp_naming_rule.uobject_prefixed.symbols					= uobject_class
+cpp_naming_rule.uobject_prefixed.style						= uobject_style
+
+cpp_naming_rule.boolelans_prefixed.symbols					= boolelan_vars
+cpp_naming_rule.boolelans_prefixed.style					= boolean_style
+
+cpp_naming_rule.structs_prefixed.symbols					= structs
+cpp_naming_rule.structs_prefixed.style						= unreal_engine_structs
+
+cpp_naming_rule.enums_prefixed.symbols						= enums
+cpp_naming_rule.enums_prefixed.style						= unreal_engine_enums
+
+cpp_naming_rule.templates_prefixed.symbols					= templates
+cpp_naming_rule.templates_prefixed.style					= unreal_engine_templates
+
+cpp_naming_rule.general_names.symbols						= all_symbols
+cpp_naming_rule.general_names.style							= unreal_engine_default
```

Comment:
If this is meant to be all aligned this and quite a few others below are showing misaligned.